analytics_data_api/v0/models.py

```diff
@@ -460,7 +460,7 @@ class ModuleEngagement(models.Model):
 
     course_id = models.CharField(db_index=True, max_length=255)
     username = models.CharField(max_length=255)
-    date = models.DateTimeField()
+    date = models.DateField()
```

Comment:
@dsjen could this have something to do with https://openedx.atlassian.net/browse/AN-7828 ? Only datetimes have the utcoffset function. Why did we have to change this field?